projectile.el

```diff
@@ -2432,6 +2433,8 @@ TEST-DIR which specifies the path to the tests relative to the project root."
                                   :compile "boot aot"
                                   :test "boot test"
                                   :test-suffix "_test")
+(projectile-register-project-type 'clj '("deps.edn")
+                                  :test-suffix "_test")
```

Comment:
as far as I know clj has no standarized way of invoking `compile` or `test` tasks (actually it has no "task" notion), so I left these undefined here.